src/main.js

```diff
@@ -218,7 +218,8 @@ function readDiscoveryFeedProvidersInDirectory(directory) {
     let systemLanguages = GLib.get_language_names();
     let settings = new Gio.Settings({ schema_id: 'com.endlessm.DiscoveryFeed' });
     let contentLanguages = settings.get_strv('force-additional-languages');
-    let languages = systemLanguages.concat(contentLanguages);
+    let wildCardLanguages = ['*'];
+    let languages = systemLanguages.concat(contentLanguages, wildCardLanguages);
```

Comment:
You can probably just push '*' directly into the array here but that point is so minor that I'm just going to merge this anyway :) Feel free to follow up.

Reply:
Thanks Sam for the review. I actually decided against adding the wild card to the array directly for readability, the other option would have been to add a comment, but the code layout with the contentLanguages array suggested the approach from above. 

PS: Sam does not like to delete branches, it's 2018, time to take risks and being brave and face fugacity :)